tox.ini

```diff
@@ -1,19 +1,26 @@
 [tox]
 minversion = 3.10
 envlist =
-  py{36,37}-sphinx{18}
-  py{36,37}-sphinx{20,21}
+  py{36,37,38,39}-sphinx{18,20,21,22,23,24,30,31,32,33,latest}
   docs
 skipsdist = True
 
 [testenv]
 description = run the whole test suite
 deps =
   .
-  pytest-selenium==1.17.0
+  pytest-selenium==2.0.1
   sphinx18: Sphinx<1.9
   sphinx20: Sphinx<2.1
   sphinx21: Sphinx<2.2
+  sphinx22: Sphinx<2.3
+  sphinx23: Sphinx<2.4
+  sphinx24: Sphinx<2.5
+  sphinx30: Sphinx<3.1
+  sphinx31: Sphinx<3.2
+  sphinx32: Sphinx<3.3
+  sphinx33: Sphinx<3.4
+  sphinxlatest: Sphinx
```

Comment:
I think we can probably scope this down a bit and only test against the latest release of each version? Similar to python: `sphinx1, sphinx2, sphinx3`?

Reply:
I noticed that there was a "half-breaking" change in 2.4 and 2.5 (the jquery change). But I'm +1 on supporting the latest versions of each mayor release. Less time waiting.